app/views/articles/_search_results.html.erb

```diff
@@ -2,6 +2,11 @@
 <div class="breadcrumb row">
   <div class="col-md-12">
     <%= render 'constraints' %>
+    <% if ! current_user.blank? %>
+      <%= "SU affiliations: #{current_user.affiliations} "%> 
```

Comment:
do we want this to be visible to users?

Reply:
This is for testing. I will remove this before I undraft the PR.